src/include/processor/operator/persistent/reader/parquet/interval_column_reader.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "common/types/interval_t.h"
+#include "templated_column_reader.h"
+
+namespace kuzu {
+namespace processor {
+
+struct IntervalValueConversion {
+    static inline common::interval_t dictRead(
+        ByteBuffer& dict, uint32_t& offset, ColumnReader& /*reader*/) {
+        return (reinterpret_cast<common::interval_t*>(dict.ptr))[offset];
```

Comment:
When will this line be triggered? Codecov seems to think this line is not covered.

Reply:
This will be triggered when the parquet file is dictionary encoded